### PR TITLE
Create LICENSE

### LINKED ISSUE #162: LICENSE file is missing in the github repo
We had the license mentioned on the www.bettercrypto.org website but not on the github page.
So far we used: https://creativecommons.org/licenses/by-sa/3.0/

### PR DESCRIPTION
**Pull request by aaronkaplan**
Fixes #162

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+This work is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License. To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/3.0/ or send a letter to Creative Commons, PO Box 1866, Mountain View, CA 94042, USA.
```
Comment (hartwork) [inline LICENSE]: Leading single space looks unintended to me. Is it intended?

Comment (aaronkaplan) [reply LICENSE]: Nope, weird. My branch did not have that extra space.
Anyway, reformatted now.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,3 @@
+This work is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License. 
+To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/3.0/ or
```
Comment (hartwork) [inline LICENSE]: Wait! https!

Comment (aaronkaplan) [reply LICENSE]: done. if you can confirm, I'll merge. thx!

**Pull request merged**